resource-server/okta-aspnetcore-webapi-example/Controllers/MessagesController.cs

```diff
@@ -2,32 +2,36 @@
 using System.Linq;
 using System.Security.Claims;
 using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Cors;
 using Microsoft.AspNetCore.Mvc;
 
+#pragma warning disable SA1300 // Element should begin with upper-case letter
 namespace okta_aspnetcore_webapi_example.Controllers
+#pragma warning restore SA1300 // Element should begin with upper-case letter
 {
     [Produces("application/json")]
     [Authorize]
     public class MessagesController : Controller
     {
         [HttpGet]
         [Route("~/api/messages")]
+        [EnableCors("AllowAll")]
```

Comment:
Did we determine if we actually need both this and the `app.UseCors("AllowAll");` in startup.cs?

Reply:
Good call! ```app.UseCors("AllowAll")``` applies the CORS policy to every request. I'm gonna push an update for this.